web/src/views/StatKeeper/RecordStats.tsx

```diff
@@ -35,6 +34,9 @@ const RecordStats: React.FC<{ bookkeeper: Bookkeeper }> = ({ bookkeeper }) => {
   const fullHomeRoster = bookkeeper.getHomeRoster();
   const fullAwayRoster = bookkeeper.getAwayRoster();
 
+  // Fetch team data to get gender information
+  const { allLeaguePlayers } = useTeams(bookkeeper.league.id.toString());
```

Comment:
feels like a workaround to get the full list here, also is this working for subs? don't see how it could. We need to carry the full player object all the way through